extensions/binding/org.eclipse.smarthome.binding.lifx/src/main/java/org/eclipse/smarthome/binding/lifx/internal/fields/HSBK.java

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.binding.lifx.internal.fields;
+
+import static org.eclipse.smarthome.binding.lifx.internal.LifxUtils.*;
+
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.library.types.HSBType;
+import org.eclipse.smarthome.core.library.types.PercentType;
+
+/**
+ * @author Wouter Born - Add support for MultiZone light control
+ */
+public class HSBK {
+
+    private static final String DEFAULT_PROPERTY_NAME = "hsbk";
+
+    private int hue;
+    private int saturation;
+    private int brightness;
+    private int kelvin;
+
+    public HSBK(int hue, int saturation, int brightness, int kelvin) {
```

Comment:
I've rewritten the copy constructor. If kept the `hsb` `temperature` constructor since it is also used in `LifxBindingConstants`. Rewriting that would again introduce some of the copy/paste code that I wanted to get rid of by doing all the conversions from ESH to LIFX colors in the `HSBK` class.